app/uk/gov/hmrc/transitmovementsauditing/models/AuditType.scala

```diff
@@ -27,7 +29,34 @@ object Sources {
 
 object AuditType {
 
-  val values: Seq[AuditType] = Seq(
+  val auditFailedEvent: Seq[AuditType] = Seq(
+    TraderFailedUploadEvent,
+    SubmitArrivalNotificationFailedEvent,
+    SubmitDeclarationFailedEvent,
+    ValidationFailedEvent,
+    CreateMovementDBFailedEvent,
+    PushNotificationFailedEvent,
+    AddMessageDBFailedEvent,
+    PushNotificationUpdateFailedEvent,
+    SubmitAttachMessageFailedEvent,
+    GetMovementsDBFailedEvent,
+    GetMovementDBFailedEvent,
+    GetMovementMessagesDBFailedEvent,
+    GetMovementMessageDBFailedEvent,
+    PushPullNotificationGetBoxFailedEvent,
+    CustomerRequestedMissingMovement,
+    NCTSRequestedMissingMovement
+  )
+
+  val auditSuccessEvent: Seq[AuditType] = Seq(
+    TraderToNCTSSubmissionSuccessful,
+    NCTSToTraderSubmissionSuccessful
+  )
+
+  val auditWorkFlowEvent: Seq[AuditType] = Seq(LargeMessageSubmissionRequested)
```

Comment:
Are these necessary? We're never going to need to query these, so why increase the maintenance burden by doing this?